src/packages/migrate/src/utils/getGithubIssueUrl.ts

```diff
@@ -0,0 +1,85 @@
+import newGithubIssueUrl from 'new-github-issue-url'
+import open from 'open'
+import prompt from 'prompts'
+import stripAnsi from 'strip-ansi'
+import {getPlatform} from '@prisma/get-platform'
+export function getGithubIssueUrl({
+  title,
+  user = 'prisma',
+  repo = 'migrate',
+  template = 'bug_report.md',
+  body,
+}: {
+  title: string
+  user?: string
+  repo?: string
+  template?: string
+  body?: string
+}): string {
+  return newGithubIssueUrl({
+    user,
+    repo,
+    template,
+    title,
+    body,
+  })
+}
+interface IssueOptions {
+  title?: string,
+  error: any,
+  cliVersion: string,
+  binaryVersion: string,
+  reportId?: number,
+}
+export async function wouldYouLikeToCreateANewIssue(
+  options: IssueOptions
+) {
+  const shouldCreateNewIssue = await prompt({
+    type: 'select',
+    name: 'value',
+    message: 'To help us even more, would you like to create an issue?',
+    initial: 0,
+    choices: [
+      {
+        title: 'Yes',
+        value: true,
+        description: `Create a new GitHub issue`,
+      },
+      {
+        title: 'No',
+        value: false,
+        description: `Don't create a new GitHub issue`,
+      },
+    ],
+  })
+  if (shouldCreateNewIssue.value) {
+    const platform = await getPlatform()
+    
+    const url = getGithubIssueUrl({
+      title: options.title ?? '',
+      body: issueTemplate(platform, options),
+    })
+    await open(url)
+  }
+}
+
+const issueTemplate = (platform: string, options: IssueOptions) => {
+    return stripAnsi(`
+Hi Prisma Team! Prisma Migrate just crashed. ${options.reportId ? `This is the report:
+  Report Id: ${options.reportId}` : ''}
+
+## Versions
+      
+| Name        | Version            |
+|-------------|--------------------|
+| Platform    | ${process.platform.padEnd(19)}| 
```

Comment:
Looks like you are using `process.platform` and not `platform`, is that intended?

Reply:
Nope, Good catch